docs/DeveloperGuide.md

```diff
@@ -1,38 +1,275 @@
+
+## Table of Contents
+1. [Acknowledgements](#acknowledgements)
+2. [Setting up, getting started](#setting-up-getting-started)
+3. [Design](#design)
+   - [Command Parsing](#command-parsing)
+   - [Ingredient Class](#ingredient-class)
+   - [Recipe Class](#recipe-class)
+4. [Implementation](#implementation)
+   - [Add and Remove Command](#add-and-remove-command)
+   - [Recipe Detail Command](#recipe-detail-command)
+   - [Recipe Possible Command](#recipe-possible-command)
+   - [Recipe All Command](#recipe-all-command)
+   - [Storage Feature](#storage-feature)
+5. [Documentation, logging, testing, configuration, dev-ops](#documentation-logging-testing-configuration-dev-ops)
+6. [Appendix: Requirements](#appendix-requirements)
+   - [Product scope](#product-scope)
+   - [User stories](#user-stories)
+   - [Use cases](#use-cases)
+   - [Non-Functional Requirements](#non-functional-requirements)
+   - [Glossary](#glossary)
+7. [Appendix: Instructions for manual testing](#appendix--instructions-for-manual-testing)
+
 
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+###### [Back to table of contents](#table-of-contents)
+
+## Setting up, getting started
+
+1. Java 11 is required for Meal Companion to function properly, please make sure your device has Java 11 installed. If not, you may download it [here](https://www.oracle.com/sg/java/technologies/downloads/#java11). If you need further asssistance installing Java 11, you may refer to the [Installation Guide](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html).
+
+2. Once you have ascertained that your device has Java 11 installed, you may download the `.jar file` [here](https://github.com/AY2223S2-CS2113T-T09-3/tp/releases/tag/v2.0). 
+
+![jar2.png](images/jar2.png)
+
+Click on `mealcompanion.jar` to automatically download the file. Place the file in a folder of your choice.
+
+If you wish to view the source code, you may click on either `Source code (zip)` or `Source code (tar.gz)` to download the source code of Meal Companion for your perusal and testing. Unzip the folder and open the project using any Integrated Development Environment (IDE), although we recommend using IntelliJ Idea. You can download IntelliJ Idea [here](https://www.jetbrains.com/idea/download/#section=windows). Please ensure you download the right version for your respective operating system. 
+
+3. Open the command terminal and navigate to the folder where you downloaded `mealcompanion.jar` using `cd <file path>` commands as necessary.
+4. Ensure the terminal is within the correct folder.
+5. Use the command `java -jar mealcompanion.jar` to launch Meal Companion.
+![startup1.png](images/startup1.png)
```

Comment:
This picture doesn't seem to be rendered on the developer guide, you might want to check whether there is error in the path?